jest.config.js

```diff
@@ -7,5 +7,5 @@ module.exports = {
   resetMocks: true,
   resetModules: true,
   restoreMocks: true,
-  testPathIgnorePatterns: ['node_modules', 'e2e'],
+  testPathIgnorePatterns: ['node_modules', '<rootDir>/e2e'],
```

Comment:
Shouldn't this be `<rootDir>/test/e2e`?

Reply:
no those are the tests i want to run here, it's the top-level `e2e/` i want to ignore. those are handled separately here: https://github.com/autoblocksai/javascript-sdk/blob/ff44b5074fe79b415d13e1b491633355a76b69bf/.github/workflows/ci.yml#L59-L87

kinda confusing, i'll attempt to clean up at some point